slf4cats-api/src/main/scala/slf4cats/api/slf4cats-api.scala

```diff
@@ -3,35 +3,24 @@ package slf4cats.api
 import cats.effect.Sync
 import org.slf4j.Marker
 
-trait LoggingContext[F[_]] {
+trait StructuredLoggerArguments[F[_]] {
   type Self <: LoggingContext[F]
```

Comment:
The `Self` needs to be renamed too
```suggestion
  type Self <: ArgumentsBuilder[F]
```